src/stores/store.ts

```diff
@@ -0,0 +1,5 @@
+import { defineStore } from "pinia";
+
+export const useCounterStore = defineStore("store", {
```

Comment:
Probably want to rename the exported store to something other than useCounterStore. 

Maybe if you create an auth.ts, do useAuthStore?